bin/stats.py

```diff
@@ -63,7 +64,7 @@ def stats(problems):
             format_string += ' {:>' + str(width + len(Fore.WHITE) + len(Style.RESET_ALL)) + '}'
 
     header = header_string.format(*headers)
-    print(Style.BRIGHT + header + Style.RESET_ALL)
+    print(Style.BRIGHT + header + Style.RESET_ALL, file=sys.stderr)
```

Comment:
The output of the `stats` command would also be printed to stdout, right? (https://github.com/RagnarGrootKoerkamp/BAPCtools/issues/92#issuecomment-752060134)

Reply:
I figured there's no point currently. All stylized output (bold/colours) won't be easily parseable anyway, so there's no point in having it on stdout.